blaze-client/src/test/scala/org/http4s/client/blaze/BlazePooledHttp1ClientSpec.scala

```diff
@@ -1,7 +1,31 @@
-package org.http4s.client.blaze
+package org.http4s
+package client
+package blaze
 
-import org.http4s.client.ClientRouteTestBattery
+import scalaz.concurrent.Task
 
+class BlazePooledHttp1ClientSpec
+  extends { val client = PooledHttp1Client() }
+    with ClientRouteTestBattery("Blaze PooledHttp1Client", client) {
 
-class BlazePooledHttp1ClientSpec extends ClientRouteTestBattery("Blaze PooledHttp1Client", PooledHttp1Client())
+  val path = "/simple"
```

Comment:
Another option would be to add `GetRoutes.simplePath(): String` or similar.  But IMO it's a tiny nit and not worth holding up lots of good changes for.